packages/bun-uws/src/HttpContext.h

```diff
@@ -534,7 +535,8 @@ struct HttpContext {
 
     /* Listen to unix domain socket using this HttpContext */
     us_listen_socket_t *listen_unix(const char *path, size_t pathlen, int options) {
-        auto* socket =  us_socket_context_listen_unix(SSL, getSocketContext(), path, pathlen, options, sizeof(HttpResponseData<SSL>));
+        int error = 0;
```

Comment:
Where is this error propagated?

Reply:
this would change uWS I think we should but in a followup